airbyte-webapp/src/pages/SettingsPage/SettingsPage.tsx

```diff
@@ -76,6 +77,11 @@ const SettingsPage: React.FC<SettingsPageProps> = ({ pageConfig }) => {
           name: <FormattedMessage id="settings.metrics" />,
           component: MetricsPage,
         },
+        {
```

Comment:
Migration is oss specific feature, right?

Reply:
No. The cloud can also have migrations. It may even migrations for extra databases that only exist for the cloud.